config/circleci

```diff
@@ -57,6 +55,10 @@ DARK_CONFIG_COOKIE_DOMAIN=.darklang.localhost
 DARK_CONFIG_USER_CONTENT_HOST=builtwithdark.localhost:8000
 DARK_CONFIG_BWDSERVER_HOST=builtwithdark.localhost:11001
 
+# Serialization
+DARK_CONFIG_SERIALIZATION_GENERATE_TEST_DATA=n
+DARK_CONFIG_SERIALIZATION_CHECK_TYPES=y
```

Comment:
Some minor commentary around these would be useful here.
`DARK_CONFIG_SERIALIZATION_CHECK_TYPES` could also be renamed to `DARK_CONFIG_SERIALIZATION_ENFORCE_TYPE_SAFETY`

Reply:
Good idea. I added some comments in LibBackend/Config.fs